conan/tools/files/files.py

```diff
@@ -164,13 +165,21 @@ def download(conanfile, url, filename, verify=True, retry=None, retry_wait=None,
     download_cache = config["tools.files.download:download_cache"] if checksum else None
 
     def _download_file(file_url):
-        # The download cache is only used if a checksum is provided, otherwise, a normal download
-        run_downloader(requester=requester, output=out, verify=verify, download_cache=download_cache,
-                       user_download=True, url=file_url,
-                       file_path=filename, retry=retry, retry_wait=retry_wait, overwrite=overwrite,
-                       auth=auth, headers=headers, md5=md5, sha1=sha1, sha256=sha256)
+        if file_url.startswith('file:'):
+            filepath = _path_from_file_uri(file_url)
+            shutil.copyfile(filepath, filename)
```

Comment:
I still wonder if this is the best place to do it - we'd have to ignore `retry`, `retry_wait`, `auth` and `headers`, but still have to check the checksums and overwrite! (working on it!)